src/MUSI6106Exec/MUSI6106Exec.cpp

```diff
@@ -1,131 +1,311 @@
 
 #include <iostream>
-#include <ctime>
+#include <cassert>
+#include <chrono>
```

Comment:
FYI: This doesn't compile on my system because `strcmp` and `memset` aren't defined. (`#include <cstring>` resolves this.)

Reply:
Fixed in 04995c5937e11713dd3747ea65f37650d2eec19f.